tests/e2e/forms/repeat-form.wdio-spec.js

```diff
@@ -0,0 +1,94 @@
+const fs = require('fs');
+
+const constants = require('../../constants');
+const utils = require('../../utils');
+const auth = require('../../auth')();
+const loginPage = require('../../page-objects/login/login.wdio.page');
+const commonPage = require('../../page-objects/common/common.wdio.page');
+const reportsPage = require('../../page-objects/reports/reports.wdio.page');
+
+const xml = fs.readFileSync(`${__dirname}/../../../demo-forms/repeat-translation.xml`, 'utf8');
+const formDocument = {
+  _id: 'form:repeat-translation',
+  internalId: 'repeat-translation',
+  title: 'Repeat',
+  type: 'form',
+  _attachments: {
+    xml: {
+      content_type: 'application/octet-stream',
+      data: Buffer.from(xml).toString('base64')
+    }
+  }
+};
+const userContactDoc = {
+  _id: constants.USER_CONTACT_ID,
+  name: 'Jack',
+  date_of_birth: '',
+  phone: '+64274444444',
+  alternate_phone: '',
+  notes: '',
+  type: 'person',
+  reported_date: 1478469976421,
+  parent: {
+    _id: 'some_parent'
+  }
+};
+
+describe('RepeatForm', () => {
+  before(async () => {
+    await utils.seedTestData(userContactDoc, [formDocument]);
+  });
+
+  afterEach(async () => {
+    await browser.deleteCookies();
+    await browser.refresh();
+  });
+
+  /* eslint-disable max-len */
+  const stateLabelPath = '#report-form .question-label.active[data-itext-id="/repeat-translation/basic/state_1:label"]';
```

Comment:
To avoid needing to disable eslint line length rule (which is there for readability), can you save the common prefix  `#report-form .active` in a separate variable and just do: 
```suggestion
const selectorPrefix = '#report-form .active';
const stateLabelPath = `${selectorPrefix}[data-itext-id="/repeat-translation/basic/state_1:label"]`;
```